### PR TITLE
add proto files

### PR DESCRIPTION
**Pull request by leeyr338**
add proto files

### CODE REVIEW

**Review by yejiayu — APPROVED** [review]
LGTM

**Comment by u2** [comment]
下次记得 format 下格式

File: protos/snapshot.proto [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    uint64 end_height = 3;
+    string file = 4;
+}
+
+
+message SnapshotResp {
+    Resp resp = 1;
+}
+
+
```
Comment (zhangsoledad) [inline protos/snapshot.proto:31]: trim empty line

File: protos/sync.proto [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+import "blockchain.proto";
+
+message SyncRequest {
+    repeated uint64 heights = 1;
+}
+
+message SyncResponse {
+    repeated Block blocks = 1;
+}
```
Comment (zhangsoledad) [inline protos/sync.proto:12]: feed new line
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

**Pull request merged**